2748. Number of Beautiful Pairs/solution.py

```diff
@@ -0,0 +1,31 @@
+class Solution:
+    # This takes in a list of nums and returns each beautiful pair,
+    # where a beautiful pair is when the first digit of the first
+    # num in the pair is coprime with the last digit of the second
+    # num in the pair. Two nums are coprime if their gcd is no greater
```

Comment:
"Two nums are coprime if their gcd is no greater than 1." Could be simplified as, "Two nums are coprime if their gcd is 1."